odbc/src/util.rs

```diff
@@ -65,13 +65,15 @@ pub fn set_error_message(
     unsafe {
         // Check if the entire error message plus a null terminator can fit in the buffer;
         // we should truncate the error message if it's too long.
```

Comment:
The change here is to remove an extra copy. It isn't necessary for asan, it's for efficiency.